doc-content/enterprise-only/project-data/preview-editor-latest-version-test-proc.adoc

```diff
@@ -0,0 +1,14 @@
+[id='preview-editor-latest-version-test-proc']
+= Switching between versions of a test scenario
+
+{CENTRAL} provides you the ability to switch between the various versions of a test scenario. Every time you save the scenario, a new version of the scenario is listed under *Latest Versions*. To use this feature you should save the test scenario file at least once.
+
+.Procedure
+. From the *Test Scenario (Preview)* editor toolbar on the upper-right, click *Latest Version*. All the versions of the file will be listed under *Latest Version* if they exist.
```

Comment:
"All the versions of the file will be listed under *Latest Version* if they exist."
Avoid "will be".
Also, this sentence should is a result of the preceding one, hence should come in the next line.